MVP.md

```diff
@@ -1,27 +1,28 @@
 # RAP Benefits
 
 A RAP should:
-- Improve the quality of the statistics.
-- Increase trust in the statistics by producers, their managers and users.
+- Improve the quality of the analysis.
+- Increase trust in the analysis by producers, their managers and users.
 - Create a more efficient process.
 - Improve business continuity and knowledge management.
 
 # RAP MVP
 
 In order to achieve these benefits, at a minimum a RAP must:
+- Augment extant technical and quality assurance processes with **peer review** to ensure that the process is reproducible and that the below requirements have been met.
 - **Minimise manual steps**, for example copy-paste, point-click or drag-drop steps. Where it is absolutely necessary to include a manual step in the process this must be documented as described below. 
-- Be built using **open source software** which is available to anyone.
-- Guarantee an **audit trail** using version control software.
+- Be built using software where the code is available to view internally, but preferably using **open source software** which is available to anyone.
+- Guarantee an **audit trail** using version control software, preferably Git.
```

Comment:
This would be a slightly different approach to the other requirements, where we haven't specific a preference for any tools. Should we add that in - for example giving a preference for R and python in the requirement about open source software?

Reply:
Yes, it's difficult to know where to draw the line. Git is pretty much the de-facto VCS these days. I don't think there's anything wrong with stating preferences, as long as they don't become mandatory.

Reply:
I agree, I think it's helpful to add preferences without making specifics mandatory here